examples/userapi/02_apply.ipynb

```diff
@@ -449,14 +453,14 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "Operator `Kernel` run in 0.00 s\n"
+      "Operator `Kernel` ran in 0.01 s\n"
      ]
     },
     {
      "data": {
       "text/plain": [
-       "PerformanceSummary([('section0',\n",
-       "                     PerfEntry(time=3e-06, gflopss=0.021333333333333333, gpointss=0.010666666666666666, oi=0.16666666666666666, ops=2, itershapes=[(2, 4, 4)]))])"
+       "PerformanceSummary([(PerfKey(name='section0', rank=None),\n",
```

Comment:
are we systematically displaying `rank`?

Reply:
Yes